supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
```

Comment:
I believe that mod docs guidance now is to use the form [id="links_{context}"].

Reply:
It does, but for now, this guide uses this type of anchor. We currently use this format now because of the legacy glossary files that were brought over. So we can definitely fix it, but it needs to be done across the entire repo.